src/EB5130.c

```diff
@@ -7,4 +7,9 @@
 
 #include "world/common/DeadUnkTexturePanFunc2.inc.c"
 
-INCLUDE_ASM(s32, "EB5130", func_8024030C_EB543C);
+extern s16 D_80169B12;
+
+ApiStatus func_8024030C_EB543C(Evt* script, s32 isInitialCall) {
+    evt_set_variable(script, *script->ptrReadPos, D_80169B12);
+    return ApiStatus_DONE2;
+}
```

Comment:
There are 3 duplicates of these functions (including one that has already been decompiled `func_80240388_EA7358` in `EA7320.c`) so deduplicating those will probably be a thing I will do next, especially since they all access `D_80169B12`. You can keep it like this though, that is something I have to worry about!